RELEASE_NOTES.md

```diff
@@ -1,5 +1,6 @@
 # Next release
 
+- [#420](https://github.com/IAMconsortium/pyam/pull/420) Refactor `data` object to a pandas.Series 
```

Comment:
Has support for datetime times also been dropped (or were the deleted tests redundant)?

```suggestion
- [#420](https://github.com/IAMconsortium/pyam/pull/420) Add `pyam.IamDataFrame._data` to handle data internally. Make `pyam.IamDataFrame._data` a `pandas.Series` to improve performance (greatly reduces duplication of metadata compared to a long dataframe)
```

Reply:
datetime support has absolutely not been dropped (though having a tutorial for it would be nice to make it easier to advertise this feature...)

The tests were removed per our email conversation on July 23 where you confirmed that they are redundant

Reply:
Re the suggestion, not sure if this is the best place - will add this extended description somewhere else in the release notes